src/components/common/FeatureBox/FeatureBox.module.scss

```diff
@@ -1,16 +1,16 @@
 @import "../../../styles/settings.scss";
 
 .root {
-  @extend %hover-effect;
-  border: 1px solid #b6b6b6;
+  //@extend %hover-effect;
+  border: 1px solid $featureBox-root-border;
   text-align: center;
   margin-top: 40px;
   width: 100%;
 
   .iconWrapper {
     height: 60px;
     transform: translateY(-50%);
-    color: $text-color;
+    color: $featureBox-root-icon-color;
```

Comment:
Tu był wcześniej kolor `#2a2a2a` zmieniłaś to teraz na biały w efekcie schowałaś ikony w sekcji FeatureBox

![image](https://user-images.githubusercontent.com/7238770/95943704-d315ea00-0de6-11eb-8222-a0e6c894da2b.png)

Reply:
racja, poprawiam, po prostu juz mi sie wszystko zaczelo mieszkac przy tylu zmiennych z kolorami "shit" :)